ee/modules/021-cni-cilium/templates/egress-gateway-agent/daemonset.yaml

```diff
@@ -42,6 +42,8 @@ spec:
       maxUnavailable: 1
   template:
     metadata:
+      annotations:
+        {{ include "helm_lib_ds_eviction_annotation" (list . "false" ) | nindent 8 }}
```

Comment:
I think func should be like `include "helm_lib_ds_eviction_annotation" (list .)` and second parameter should be optional (false by default) or second parameter should be absent - we do not need annotation with value `true` because it is default behavior.

Reply:
Fixed, also restored scale from zero patches